README.md

```diff
@@ -1,10 +1,18 @@
 # Exercism Groovy Track
-
+[![Build Status](https://travis-ci.org/exercism/groovy.svg?branch=master)](https://travis-ci.org/exercism/groovy)
+[![GitHub issues](https://img.shields.io/github/issues/exercism/groovy.svg)](https://github.com/exercism/groovy/issues)
+[![GitHub forks](https://img.shields.io/github/forks/exercism/groovy.svg)](https://github.com/exercism/groovy/network)
+[![GitHub license](https://img.shields.io/badge/license-MIT-blue.svg)](https://raw.githubusercontent.com/exercism/groovy/master/LICENSE)
 [![Join the chat at https://gitter.im/exercism/groovy](https://badges.gitter.im/exercism/groovy.svg)](https://gitter.im/exercism/groovy?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
 
 Exercism problems in Groovy
 
 ## Contributing Guide
+If you want to be a part of Excercism as a whole and contribute, please refer to the [Contributing Guide](https://github.com/exercism/docs/blob/master/finding-your-way.md).
```

Comment:
I'm confused and can't understand where exactly the typo is.
Could you please clarify?